pages/Hardened-Glass.md

```diff
@@ -0,0 +1,7 @@
+Hardened Glass is a [Technical Component](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Technical-Components) that can be used to craft a variety of other Technical Components.
+
+## Obtaining
+Hardened Glass can be crafted in the [Enhanced Crafting Table](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Enhanced-Crafting-Table)
+
+## Special Properties
+Hardened Glass is resistant to most explosions.
```

Comment:
Maybe mention that it is not resistant to Withers, but the wither-proof glass is , in a [see also].

Reply:
Added that it is not immune to wither explosions, also added a See Also to Wither-Proof Blocks.